develop/parachain-devs/parachain-maintenance/runtime-upgrades.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Runtime Upgrades
+description: Explains how runtime versioning and storage migration support forkless upgrades for Polkadot SDK-based networks and they factor in to chain upgrades.
+---
+
+# Runtime Upgrades
+
+The on-chain forkless runtime upgrades are one of the defining features of Polkadot SDK-based blockchains. The ability to update the runtime logic without forking the network enables your blockchain to evolve and improve seamlessly over time. This is made possible through the [Wasm](https://webassembly.org/){target=\_blank} powered runtimes that are stored in the blockchain state. When upgrading the blockchain network, the Wasm code containing the new runtime logic is uploaded on-chain and is swapped with the old code when the upgrade is enacted.
```

Comment:
I think this intro could benefit from the [formula for intros](https://github.com/papermoonio/documentation-style-guide/blob/main/style-guide.md#introductions) defined in the style guide

We could remove "Why Forkless" from the info admonition and have that be part of describing the current standard.

Reply:
There's a lot of content here, so some of it may need to be broken up 🤷‍♀️